src/main/java/uk/nhs/hee/tis/common/upload/controller/AwsStorageController.java

```diff
@@ -25,4 +35,28 @@ public ResponseEntity uploadFile(@ModelAttribute @Valid FileUploadDto fileUpload
     final var response = awsStorageService.upload(fileUpload);
     return ResponseEntity.ok(response);
   }
+
+  @GetMapping
+  public ResponseEntity downloadFile(@RequestParam("bucketName") final String bucketName,
+      @RequestParam("folderPath") final String folderPath,
+      @RequestParam("fileName") final String fileName) {
```

Comment:
Will the file name be unique? Do we need some unique variable like file id to identify the file?

Reply:
why we need a file Id?